packages/roosterjs-editor-types/lib/interface/IEditor.ts

```diff
@@ -499,6 +499,13 @@ export default interface IEditor {
      */
     getStyleBasedFormatState(node?: Node): StyleBasedFormatState;
 
+    /**
+     * Get the pendable format such as underline and bold
+     * @param forceGetStateFromDOM If set to true, will force get the format state from DOM tree.
```

Comment:
Also explain what is the default behavior.

Reply:
We don't need to mention the cache thing in the comment for this whole function, but we can explain how it works for this parameter, so explain cache here is ok